apps/docs/src/components/Hero/Hero.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import Link from '@docusaurus/Link'
+import Cover from '/img/newperf.png'
+
+function Hero() {
+  return (
+    <div className='bg-whiteIce mb-24'>
+      <div className="mx-auto p-4 w-full lg:max-w-[1170px] lg:py-16">
+        <div className="flex justify-between flex-col md:flex-row">
+          <div className="my-auto md:pr-24">
+            <h1 className='font-VTEXTrust text-sm text-details'>FastStore</h1>
+            <h2 className="text-[52px] text-fontSecondary font-VTEXTrust leading-tight">
+            The ultimate toolkit for building blazing-fast storefronts
+            </h2>
+            <p className="text-base text-details my-6 leading-7">
+            In the ecommerce world, speed matters. That's why we've built FastStore: an open-source toolkit to create blazing-fast storefronts that convert.
+            </p>
+            <div>
+              <div>
+                <Link
+                  to="https://www.vtexfaststore.com/"
+                  className="button-primary"
+                >
+                  VIEW LIVE DEMO
+                </Link>
+                <Link
+                  to="https://www.vtexfaststore.com/"
```

Comment:
```suggestion
                  to="https://content.vtex.com/vtexio-headless-cms-closed-beta/"
```

Isso aqui para de fato mandar para o site onde dá pra join the waiting list